pranav/cart/spec.rb

```diff
@@ -0,0 +1,123 @@
+require "./main.rb"
+
+describe Cart do
+  describe "#cal_gst"
+    context "calculate GST on gross_amount" do      
+      before do
+        s_gst = 8
+        c_gst = 8
+      end      
+      it "return GST amount on gross_amount" 
+        true_amount = (gross_amount / 100) * (c_gst + s_gst)
+        gst_amount = cal_gst(amount)
+        expect(gst_amount).to eq(true_amount)
+      end
+    end  
+  end  
+end 
```

Comment:
have you executed your test cases?
please attach output of the same.

Reply:
I had to refactor my code and for separating logic of Cart and Menu. Screen shot for Test cases of Menu is added below.  
file:///home/pranav/kylas/learning/cart/ruby_Kylas/pranav/cart/ss/Screenshot%20from%202023-03-03%2011-19-45.png